integrationtest/src/main/scala/io/gearpump/integrationtest/minicluster/MiniCluster.scala

```diff
@@ -110,6 +110,10 @@ class MiniCluster(
     } != "")
   }
 
+  def getDockerMachineIp(): String = {
+    Docker.execAndCaptureOutput(MASTER_ADDRS.head._1, "ip route").split("\\s+")(2)
```

Comment:
```
manu@manu-IdeaCentre-K410:~/git/gearpump$ docker exec -it master0 /bin/sh
/opt/gearpump # ip route
default via 172.17.42.1 dev eth0 
172.17.0.0/16 dev eth0  src 172.17.0.223 
/opt/gearpump # manu@manu-IdeaCentre-K410:~/git/gearpump$ ifconfig
docker0   Link encap:Ethernet  HWaddr 56:84:7a:fe:97:99  
          inet addr:172.17.42.1  Bcast:0.0.0.0  Mask:255.255.0.0
          inet6 addr: fe80::5484:7aff:fefe:9799/64 Scope:Link
          UP BROADCAST RUNNING MULTICAST  MTU:1500  Metric:1
          RX packets:1254282 errors:0 dropped:0 overruns:0 frame:0
          TX packets:1521375 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:0 
          RX bytes:137076832 (137.0 MB)  TX bytes:203465612 (203.4 MB)
```

`ip route` inside container gives us `default via 172.17.42.1` and `172.17.42.1` is exactly the ip address of `docker0` on the host.